modules/partition-replicator/src/main/java/org/apache/ignite/internal/partition/replicator/network/PartitionReplicationMessageGroup.java

```diff
@@ -218,6 +220,16 @@ public interface PartitionReplicationMessageGroup {
      */
     short CHANGE_PEERS_AND_LEARNERS_ASYNC_REPLICA_REQUEST = 28;
 
+    /**
+     * Message type for {@link GetEstimatedSizeWithLastModifiedTsRequest}.
+     */
+    short GET_ESTIMATED_SIZE_WITH_MODIFIED_TS_MESSAGE_REQUEST = 29;
+
+    /**
+     * Message type for {@link GetEstimatedSizeWithLastModifiedTsResponse}.
+     */
+    short GET_ESTIMATED_SIZE_WITH_MODIFIED_TS_MESSAGE_RESPONSE = 30;
```

Comment:
I think, these messages should not be part of partition replication group. The rule of thumb here: you add message to this group if you will be using `ReplicaService` as client.